mlir/unittests/Dialect/QCO/IR/Modifiers/test_qco_ctrl.cpp

```
@@ -0,0 +1,214 @@
+/*
+ * Copyright (c) 2023 - 2026 Chair for Design Automation, TUM
+ * Copyright (c) 2025 - 2026 Munich Quantum Software Company GmbH
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ * Licensed under the MIT License
+ */
+
+#include "mlir/Dialect/QCO/IR/QCODialect.h"
```

Cppcheck (2.19.0)

[information] Include file

(missingInclude)